apps/server/src/modules/deletion/uc/deletion-request.uc.ts

```diff
@@ -42,7 +43,8 @@ export class DeletionRequestUc {
 		private readonly registrationPinService: RegistrationPinService,
 		private readonly filesStorageClientAdapterService: FilesStorageClientAdapterService,
 		private readonly dashboardService: DashboardService,
-		private readonly taskService: TaskService
+		private readonly taskService: TaskService,
+		private readonly newsUc: NewsUc
```

Comment:
Please don't import the `NewsUc` here - we're trying to stick to the rule to import only other services, not _use cases_. I know the `NewsService` doesn't exist yet, but please create one a put this `deleteCreatorOrUpdaterReference()` method that you've added to the `NewsUc` to this `NewsService` (with the code adjusted to the service-type object of course).

Reply:
done